avidemux_ru.ts

```diff
@@ -11475,17 +11475,17 @@ The file must be re-indexed. Proceed?</source>
     <message>
         <location line="+460"/>
         <source>Recent Files</source>
-        <translation>Предыдущие файлы</translation>
+        <translation>Недавние файлы</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Recent Projects</source>
-        <translation>Предыдущие проекты</translation>
+        <translation>Недавние проекты</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Restore defaults</source>
-        <translation>Восстановить стандартный набор</translation>
+        <translation>Сбросить выбор</translation>
```

Comment:
The default is everything enabled / selected. The user chooses to _deselect_ an option. I feel a little bit of cognitive dissonance with the (standard) wording of the new translation, would prefer the previous one.

Reply:
Okey, will be reverted